packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -16,6 +16,7 @@ import { ChartHoverCard, convertToLocaleString, getAccessibleDataObject } from '
 import { FocusZone, FocusZoneDirection } from '@fluentui/react-focus';
 import { formatPrefix as d3FormatPrefix } from 'd3-format';
 import { FocusableTooltipText } from '../../utilities/FocusableTooltipText';
+import { clamp } from '@fluentui/react';
```

Comment:
The bars are still getting cut in dark mode

Reply:
I missed this. Thanks for notifying. Will work on this

Reply:
![image](https://github.com/microsoft/fluentui/assets/70366079/fac605bb-333a-473d-9881-a6ddb827275e)